v2/fleet-local/proxy/capcom@.service

```diff
@@ -1,6 +1,6 @@
 [Unit]
-Description=Capcom/{HAProxy NGinx} @ %i
-After=docker.service
+Description=Capcom @ %i
+After=docker.service proxy@%i.service
 Requires=docker.service
 
 [Service]
```

Comment:
In this `[Service]` block I would add an `ExecStartPre=/usr/bin/systemctl is-active proxy@*`

That will explicitly block this unit from starting up if capcom is not running